lib/sir_handel.rb

```diff
@@ -141,6 +142,12 @@ def search
       }
     end
 
+    def with_trend(search)
```

Comment:
possibly, but it's only used once. Might be a sign to start dividing the web app object into more pieces.